### ISSUE TITLE
Flatten all input by default to Model.Create

### ISSUE THREAD

**Issue by xRadne** [strusoft/femdesign-api#366]
Make the input to Model.Create flatten the input by default

To avoid this:
![image](https://user-images.githubusercontent.com/32938476/168296962-d12cd7c3-5792-436b-9082-0905eaf53593.png)

**Comment by xRadne** [comment]
The components should preferably look like this when initialized on the canvas
![image](https://user-images.githubusercontent.com/32938476/168297285-4becbeb5-8c01-476e-8bb2-6b3433e99d0d.png)

**Comment by Marco-Pellegrino** [comment]
I am not totally convinced.
Most of the user will definitely want to have a flatten list in that component but it is still possible to use "Model.Create" with DataTree in order to create `n` model at the same time (it works. I have tried while I was creating some test case)

Showing a FlattenList as default will make things at some of the most expert users that they can not use DataTree.
From a Grasshopper User point of view, it is a common practice to have a container and flatten the data there.

![image](https://user-images.githubusercontent.com/53620289/168541006-074307f4-de9c-4b67-a208-d8f793b3a868.png)

**Comment by xRadne** [comment]
I disagree. If we **default** the input parameters to **flatten**, I think expert users will simply unflatten the input (if they want several models), while the novice user **will not** accidentally get stuck in an analysis loop of thousands of models with one element each 😉

I agree that the common, best practice still is the container/merge component with a flatten, but I think it is so rare for most of our users to want a list of models, that we will help more users by default to flatten all inputs.

**Comment by Marco-Pellegrino** [comment]
happy to have a flatten input by default but it would be great to let know to some of the users (probably just me and you) that it is still possible to use DataTree and generate several models.

It is even true that we are not supposed to teach the users how to use Grasshopper 😀
